src/Components/GenreSelector/GenreSelector.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+const title = 'Genre Selector';
+
+export class GenreSelector extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            selected: parseInt(props.selectedGenre),
+            genreList: props.genreList
+        };
+    }
+
+    selectGenre(genre){
+        let state = {...genre};
```

Comment:
I noticed usually you use 'const'. Is there any reason for using 'let' here?

Reply:
state would be changing, so I thought let would be appropriate for this variable declaration